### PR TITLE
ctheorems:0.1.0

### PR DESCRIPTION
**Pull request by DVDTSB**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

Description: It's useful for creating numbered environments, meant to continue and extend the classic 'typst-theorems' module, and making it available as an official package.

### CODE REVIEW

File: packages/preview/ctheorems/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = [
+    "sahasatvik (Satvik Saha)",
+    "rmolinari (Rory Molinari)",
+    "MJHutchinson (Michael Hutchinson)",
+    "DVDTSB",
+]
+license = "MIT"
+description = "The classic typst-theorems package! An implemenation of numbered theorem environments in typst."
```
Comment (laurmaedje) [inline packages/preview/ctheorems/0.1.0/typst.toml:12]:
```suggestion
description = "The classic typst-theorems package! An implementation of numbered theorem environments in Typst."
```

**Comment by laurmaedje** [comment]
If it's based on typst-theorem, there should probably be some attribution, either in a NOTICE file or by keeping the copyright of the original author in the LICENSE.

**Comment by DVDTSB** [comment]
> If it's based on typst-theorems, there should probably be some attribution, either in a NOTICE file or by keeping the copyright of the original author in the LICENSE.

Added every contributor in the README, aswell as adding the original author in the license (MIT).

File: packages/preview/ctheorems/0.1.0/typst.toml [hunk 2]
```diff
@@ -9,6 +9,6 @@ authors = [
     "DVDTSB",
 ]
 license = "MIT"
-description = "The classic typst-theorems package! An implementation of numbered theorem environments in Typst."
+description = "Theorem library based on (and compatible) with [typst-theorems](https://github.com/sahasatvik/typst-theorems)."
```
Comment (laurmaedje) [inline packages/preview/ctheorems/0.1.0/typst.toml:12]: Final nit: The descriptions aren't Markdown, so they can't contain links.

Comment (DVDTSB) [reply packages/preview/ctheorems/0.1.0/typst.toml]: fixed :D

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**